life_server/init.sqf

```diff
@@ -151,6 +151,7 @@ TON_fnc_requestClientID =
 /* Event handler for logs */
 "money_log" addPublicVariableEventHandler {diag_log (_this select 1)};
 "advanced_log" addPublicVariableEventHandler {diag_log (_this select 1)};
+"exploit_log" addPublicVariableEventHandler {diag_log (_this select 1)};
```

Comment:
Well, I mean, look at it...

All it does is diag_log what is passed, same as the others